superset/assets/src/explore/components/DisplayQueryButton.jsx

```diff
@@ -75,9 +77,12 @@ export default class DisplayQueryButton extends React.PureComponent {
       error: null,
       filterText: '',
       sqlSupported: datasource && datasource.split('__')[1] === 'table',
+      propertiesModalOpen: false,
```

Comment:
small nit: i generally prefer to start boolean attributes with `is` or `has`, so this would be `isPropertiesModalOpen`. i find it adds a bit of clarity